xml/Microsoft.Build.Framework/TaskFinishedEventArgs.xml

```diff
@@ -26,7 +26,6 @@
     <summary>
             Arguments for task finished events
             </summary>
-    <remarks>To be added.</remarks>
```

Comment:
Were these removed manually? There's no need to remove them, they won't show up on the published docs.